docs/site/DataSources.md

```diff
@@ -52,3 +52,25 @@ export class DbDataSource extends juggler.DataSource {
   }
 }
 ```
+
+### Creating a DataSource at Runtime
+
+A datasource can be created at runtime by creating an instance of
+`juggler.DataSource`. It requires a name for the datasource, the connector, and
+a connection url as shown below.
```

Comment:
I find this misleading. The configuration object accepts any valid fields, e.g. `host`, `port` and `user`. The connection url is just one of the many valid ways how to configure a datasource.

Please clarify.

I think there is no need to repeat all details here, we should instead add a link to an existing documentation page showing different datasource configuration options.

Reply:
Updated to make the statement more generic and linked to https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource.

There's no LB4 doc with details about `DataSource` options. Ideally it should be on https://loopback.io/doc/en/lb4/DataSources.html.

Reply:
> There's no LB4 doc with details about `DataSource` options. Ideally it should be on https://loopback.io/doc/en/lb4/DataSources.html.

Perhaps you can open a new issue for that and add it to "Documentation improvements" epic #5113?

Reply:
Opened yesterday - https://github.com/strongloop/loopback-next/issues/5646.